src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -200,482 +198,306 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
     return visitChildren(ctx);
   }
 
-  antlrcpp::Any visitDescribeQuery(
-      SparqlAutomaticParser::DescribeQueryContext* ctx) override {
+  antlrcpp::Any visitDescribeQuery(Parser::DescribeQueryContext* ctx) override {
+    // TODO: unsupported
```

Comment:
Can't you already implement the "visitTypesafe" method for these cases using your "throwNotSupported"-helper (I don't recall the name).